<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configwindow</class>
 <widget class="QWidget" name="configwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigManager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_19">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Gateway Power Type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="acRadioButton">
             <property name="text">
              <string>AC</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="batteryRadioButton">
             <property name="text">
              <string>Battery</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Network</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Adapater IPv4 Address:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Adapater IPv4 Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Adapater IPv6 Address:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Adapater IPv6 Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Local IPv4 Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Local IPv6 Port:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="adapterIPv4Addr"/>
              </item>
              <item>
               <widget class="QLineEdit" name="adapterIPv4Port"/>
              </item>
              <item>
               <widget class="QLineEdit" name="adapterIPv6Addr"/>
              </item>
              <item>
               <widget class="QLineEdit" name="adapterIPv6Port"/>
              </item>
              <item>
               <widget class="QLineEdit" name="localIPv4Port"/>
              </item>
              <item>
               <widget class="QLineEdit" name="localIPv6Port"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>CDMA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>CDMA Username:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>CDMA Password:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLineEdit" name="cdmaUsername"/>
              </item>
              <item>
               <widget class="QLineEdit" name="cdmaPassword">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Address</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>SEGMAC:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>SCMAC:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLineEdit" name="segmac"/>
              </item>
              <item>
               <widget class="QLineEdit" name="scmac">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Thold</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="AlertSwitc">
                <property name="text">
                 <string>Alert On/Off:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="alertOn">
                <property name="text">
                 <string>On</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="alertOff">
                <property name="text">
                 <string>Off</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Air Temp</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Air Humi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Earth Temp</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Earth Humi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>CO2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Illum</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Upper Limit</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="airTempUpper"/>
              </item>
              <item>
               <widget class="QLineEdit" name="airHumiUpper"/>
              </item>
              <item>
               <widget class="QLineEdit" name="earthTempUpper"/>
              </item>
              <item>
               <widget class="QLineEdit" name="earthHumiUpper"/>
              </item>
              <item>
               <widget class="QLineEdit" name="co2Upper"/>
              </item>
              <item>
               <widget class="QLineEdit" name="illumUpper"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Lower Limit</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="airTempLower"/>
              </item>
              <item>
               <widget class="QLineEdit" name="airHumiLower"/>
              </item>
              <item>
               <widget class="QLineEdit" name="earthTempLower"/>
              </item>
              <item>
               <widget class="QLineEdit" name="earthHumiLower"/>
              </item>
              <item>
               <widget class="QLineEdit" name="co2Lower"/>
              </item>
              <item>
               <widget class="QLineEdit" name="illumLower"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Phone Number 1:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Phone Number 2:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Phone Number 3:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <widget class="QLineEdit" name="phoneNum1">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="phoneNum2">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="phoneNum3">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Period</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Upload Period(AC):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Collect Period(Battery):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Collect Time(Battery):</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLineEdit" name="uploadPeriod"/>
              </item>
              <item>
               <widget class="QLineEdit" name="collectPeriod">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="collectTime">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>137</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Control</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>Local Control:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QRadioButton" name="localManuControl">
                    <property name="text">
                     <string>Manual</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="localAutoControl">
                    <property name="text">
                     <string>Auto</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="localControlOff">
                    <property name="text">
                     <string>Off</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="networkControlOn">
                  <property name="text">
                   <string>Network Control</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>137</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <tabstops>
  <tabstop>acRadioButton</tabstop>
  <tabstop>batteryRadioButton</tabstop>
  <tabstop>adapterIPv4Addr</tabstop>
  <tabstop>adapterIPv4Port</tabstop>
  <tabstop>adapterIPv6Addr</tabstop>
  <tabstop>adapterIPv6Port</tabstop>
  <tabstop>localIPv4Port</tabstop>
  <tabstop>localIPv6Port</tabstop>
  <tabstop>alertOn</tabstop>
  <tabstop>alertOff</tabstop>
  <tabstop>airTempUpper</tabstop>
  <tabstop>airTempLower</tabstop>
  <tabstop>airHumiUpper</tabstop>
  <tabstop>airHumiLower</tabstop>
  <tabstop>earthHumiUpper</tabstop>
  <tabstop>earthHumiLower</tabstop>
  <tabstop>co2Upper</tabstop>
  <tabstop>co2Lower</tabstop>
  <tabstop>illumUpper</tabstop>
  <tabstop>illumLower</tabstop>
  <tabstop>phoneNum1</tabstop>
  <tabstop>phoneNum2</tabstop>
  <tabstop>phoneNum3</tabstop>
  <tabstop>cdmaUsername</tabstop>
  <tabstop>cdmaPassword</tabstop>
  <tabstop>segmac</tabstop>
  <tabstop>scmac</tabstop>
  <tabstop>uploadPeriod</tabstop>
  <tabstop>collectPeriod</tabstop>
  <tabstop>collectTime</tabstop>
  <tabstop>localManuControl</tabstop>
  <tabstop>localAutoControl</tabstop>
  <tabstop>localControlOff</tabstop>
  <tabstop>networkControlOn</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
